Python/accumulate_position_estimates/accumulate_position_estimates.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Position:
+    x: float = 0
+    y: float = 0
+    z: float = 0
+
+
+pool_size = Position(25, 12.5, 5)
+
+
+class PositionEstimate:
+    def __init__(self, m_last_elem, distance_threshold) -> None:
+        self.mean = Position(0, 0, 0)
+        self.variance = Position(0, 0, 0)
+        self.measurements = [[], [], []]
```

Comment:
Is there a reason measurements are not a list of position objects?

You can extend that class to allow summation, subtraction, etc.

Reply:
I used a list of position objects at first, but it was a lot easier to find mean and variance of al x, y and z positions when I used three lists instead. 
Do you think it is better (why?) to use position objects and special methods?

Reply:
seems legit jesjes